canonical-websites/snapcraft.io
# Updated homepage

### QA

1. Pull the branch 
2. Open the homepage
3. Check the content is similar to the [copydoc]
(https://docs.google.com/document/d/1_uwPkHmAgV6_uE9ISqIgXjZGJxXPBD90vdeO95h7ikE/edit?ts=5b97a3f7#)

Note: The 'Publish from your workflow' section is not added to the homepage as there are missing docs on certain CI workflows.
